app/src/views/private/private-view.vue

```diff
@@ -409,8 +409,12 @@ const showLicenseBanner = computed(() => userStore.isAdmin && settingsStore.sett
 		font-family: var(--theme--navigation--list--font-family);
 		border-inline-end: var(--theme--navigation--border-width) solid var(--theme--navigation--border-color);
 
+		html[dir="rtl"] & {
+			transform: translateX(100%);
+		}
+
 		&.is-open {
-			transform: translateX(0);
+			transform: translateX(0) !important;
 		}
```

Comment:
Let’s prevent `!important`.

```suggestion
		html &.is-open {
			transform: translateX(0);
		}
```